Spread/AdminSpread.php

```diff
@@ -35,12 +34,13 @@ class AdminSpread implements SpreadInterface
     protected $userClass;
 
     /**
-     * @param UserManagerInterface $userManager
+     * @param RegistryInterface $registry
+     * @param UserInterface     $userClass
```

Comment:
ah no, its not typehinted, i think we then should typehint it in the constructor... confused :smile: